README.md

```diff
@@ -238,6 +238,171 @@ To run the examples in a browser using WebAssembly, you can use [wasm-server-run
+
+```rust
+fn refresh_key_bindings(mut key_bindings: ResMut<Persistent<KeyBindings>>) {
+  key_bindings.refresh();
+}
+```
+
+This system would read the settings from the disk again to be up-to-date with external modifications!
+
+- **bevy_pkv parses the objects on each read**
+
+Writes being extremely optimized, reads can be slow in [bevy_pkv](https://github.com/johanhelsing/bevy_pkv) because reading a persistent object requires parsing, on each read!
+
+In [bevy-persistent](https://github.com/umut-sahin/bevy-persistent/), reads are basically free as `Persistent<R>` is just a wrapper around the actual object. Though this comes with the drawback of memory usage, as objects are always kept in memory.
+
+Maybe [bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) can provide a solution to this problem at some point if it's requested by the community! I really like the idea of providing `.load()` and `.unload()` methods for `Persistent<R>`.
+
+```rust
+fn unload_key_bindings(mut key_bindings: ResMut<Persistent<KeyBindings>>) {
+  key_bindings.unload();
+}
+
+fn load_key_bindings(mut key_bindings: ResMut<Persistent<KeyBindings>>) {
+  key_bindings.load();
+}
+
+fn utilize_key_bindings(key_bindings: Res<Persistent<KeyBindings>>) {
+  let jump_key = key_bindings.jump; // this would panic if the resource is unloaded
+}
+```
+
+Again, always profile before making decisions for performance reasons!
+
+- **bevy_pkv doesn't do automatic management**
+
+With [bevy_pkv](https://github.com/johanhelsing/bevy_pkv), modifying the object and making the changes persistent is always two different steps.
+
+```rust
+fn modify_key_bindings(mut pkv: ResMut<PkvStore>) {
+  let mut key_bindings = pkv.get::<KeyBindings>("key-bindings");
+  key_bindings.crouch = KeyCode::LControl;
+  pkv.set("key-bindings", &key_bindings)
+}
+```
+
+[bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) on the other hand provides APIs to automate this process (see [Modification](#modification)).
+
+- **bevy_pkv makes external edits very hard (in native apps)**
+
+[bevy-persistent](https://github.com/umut-sahin/bevy-persistent/) stores each persistent object as a separate file in the specified format. If a textual format is used, the object becomes extremely easy to edit! The example in [Creation](#creation) will create `key-bindings.toml` in the specified location with the following contents:
+
+```toml
+jump = "Space"
+crouch = "C"
+```
+
+You can easily change this file to:
+
+```toml
+jump = "Space"
+crouch = "LControl"
+```
+
+And it'd work perfectly in the next run!
+
+[bevy_pkv](https://github.com/johanhelsing/bevy_pkv) on the other hand stores the objects in a single database in a binary format, which is more efficient but not suitable external modification.
+
+- **bevy_pkv makes storing objects in different locations very hard (in native apps)**
```

Comment:
Why is it hard?

Reply:
Because location of each persistent object within a `PkvStore` is not specified and you can't have `pkv_on_location1: Res<PkvStore>, pkv_on_location2: Res<PkvStore>`.

Like you cannot have `settings` and `saves` in different directories in the filesystem.

Reply:
Yes, you can. You just create a newtype for one of the stores...